newsroom/companies/views.py

```diff
@@ -127,7 +127,6 @@ def delete(_id):
     """
     Deletes the company and users of the company with given company id
     """
-    get_resource_service('users').delete_action(lookup={'company': ObjectId(_id)})
```

Comment:
Not sure what you're trying to achieve here. This stops the deletion of **ALL** users when deleting a company, which means we will have users that are not associated with a company that no longer exists.